### ISSUE TITLE
analyze neural network in the browser

### ISSUE THREAD

**Issue by blueberrynfig** [blueberrynfig/illegal-dumping-object-detection-engine#37]
http://playground.tensorflow.org/#activation=sigmoid&regularization=L1&batchSize=10&dataset=circle&regDataset=reg-plane&learningRate=0.03&regularizationRate=0.01&noise=0&networkShape=3,3,2,2,2&seed=0.35097&showTestData=false&discretize=false&percTrainData=50&x=true&y=true&xTimesY=true&xSquared=true&ySquared=true&cosX=false&sinX=false&cosY=false&sinY=false&collectStats=false&problem=classification&initZero=false&hideText=false
